index.js

```diff
@@ -1,38 +1,89 @@
+//final week 10
 import express from 'express'
 import bodyParser from 'body-parser'
 import redis from 'redis'
 
-// make the one and only express object     -   done?
 const app = express()
 
-// make a database connection to redis      -   done?
 var urlencodedParser = bodyParser.urlencoded({ extended: false})
 const db = redis.createClient({
     url: 'redis://localhost:6379'
 })
-await db.connect()     // if this errors, make sure redis is running    -   done?
+await db.connect()    
 
-// add routes
-// this route serves static files
 app.use(express.static('public'))
 
-// this route serves the home page, using code
-app.post('/add', urlencodedParser, async (req, res) => {
-    await db.set("form-data", req.body.answerText)
-    res.send(`<!DOCTYPE html><html><body>Data saved to Redis key named 'form-data'</body></html>`)
-    res.redirect()
+
+
+app.get('/', async(req, res) => {
+    res.send(`<!DOCTYPE html><html><body><A href="/home.html">Say Hello</A></body></html>`)
 })
+
+app.post('/add', urlencodedParser, async(req, res) => {
+    var quizName = req.body.quizName
+    await db.set('quizName', quizName)
+    console.log(quizName)
+    await db.hSet('questionCount', quizName, '0')
+    const g = await db.get('quizName')
```

Comment:
You should mark temporary code with a comment like `// TEMP` so others know what it is for.

Reply:
I understand your confusion now... see other comment.